### ISSUE TITLE
Markdown shows up as HTML with addon-info

### ISSUE THREAD

**Issue by SethDavenport** [storybooks/storybook#3458]
### Bug or support request summary

I'm using `@storybook/addon-info` to get some markdown into my stories, however, I'm loading it from an external file:

```js
// @flow
import React from 'react';
import { storiesOf } from '@storybook/react';
import { withInfo } from '@storybook/addon-info';
import docs from './README.md';

const withDocs = withInfo(docs);

storiesOf('A story', module)
  .add('primary', withDocs(() => <p>Some JSX</p>))
```

What I'm noticing is that it works fine if the markdown just has a header:

```md
# Header
```

![image](https://user-images.githubusercontent.com/4614141/39071096-c9e4f9ac-44b3-11e8-95ee-769517fcb0b5.png)

But as soon as I add another line, the info panel shows the first few lines as escaped HTML instead:

```md
# Header

some text
```

![image](https://user-images.githubusercontent.com/4614141/39071077-bc3ac3b8-44b3-11e8-81dc-eee8ef91269b.png)

Any idea what I'm doing wrong?

### Steps to reproduce

1. make a vanilla react storybook: no custom webpack, etc.
2. install `@storybook/addon-info`
3. use this babelrc:

```
{
  "presets": [
    "babel-preset-env",
    "react",
    "flow"
  ],
  "plugins": [
    "transform-object-rest-spread",
    "transform-async-to-generator",
    "babel-plugin-styled-components"
  ]
}
```

4. Make a story:

```js
// @flow
import React from 'react';
import { storiesOf } from '@storybook/react';
import { withInfo } from '@storybook/addon-info';
import docs from './README.md';

const withDocs = withInfo(docs);

storiesOf('A story', module)
  .add('primary', withDocs(() => <p>Some JSX</p>))
```

5. Add a header and a paragraph to the readme.md file.

### Please specify which version of Storybook and optionally any affected addons that you're running

- `@storybook/react` 3.4.2
- `@storybook/addon-info` 3.4.2

### Affected platforms 

Mac OS Sierra, Chrome 65

**Comment by Keraito** [comment]
Sorry @SethDavenport, I forgot to reply here after looking into a fix and proposing the changes. 

Basically, the problem is that the content of a `.md` file, when imported, is automatically turned into the corresponding HTML. This results in some flaky rendering. As a workaround while waiting for the PR, you can try  [using webpack to change the import to load in the file raw](https://github.com/rexxars/react-markdown/issues/76) or use an library like [turndown](https://github.com/domchristie/turndown) to turn it into proper Markdown again.

**Comment by SethDavenport** [comment]
OK yeah that makes sense.  I'll give that a shot. Thanks.

**Comment by Hypnosphi** [comment]
released as [`4.0.0-alpha.6`](https://github.com/storybooks/storybook/releases/tag/v4.0.0-alpha.6)
